src/NUnitFramework/framework/Constraints/RegexConstraint.cs

```diff
@@ -68,8 +70,9 @@ public RegexConstraint IgnoreCase
         /// <returns>True for success, false for failure.</returns>
         public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
-            return new ConstraintResult(this, actual,
-                actual is string actualString && _regex.Match(actualString).Success);
+            string actualString = ConstraintUtils.RequireActual<string>(actual, nameof(actual), allowNull: false);
+
+            return new ConstraintResult(this, actual,_regex.IsMatch(actualString));
```

Comment:
Nitpick. There should be a space after a `,`. 

Our .editorconfig shows:  `csharp_space_after_comma = true`
I thought it should have raised some warning, but it doesn't seem to.
Even adding `EnforceCodeStyleInBuild` doesn't seem to do anything.
Only enabling the StyleCop rule does so:
```none
# Commas should be spaced correctly
dotnet_diagnostic.SA1001.severity = warning
```

I will create a future PR to enable this so we get consistent code.